modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetTrafficSourcesMVCResourceCommand.java

```diff
@@ -160,53 +183,54 @@ private JSONArray _getTrafficSourcesJSONArray(
 		ResourceBundle resourceBundle) {
 
 		Map<String, String> helpMessageMap = HashMapBuilder.put(
+			"direct",
+			ResourceBundleUtil.getString(
+				resourceBundle,
+				"this-is-the-number-of-page-views-generated-by-people-" +
+					"arriving-directly-to-your-page")
+		).put(
 			"organic",
 			ResourceBundleUtil.getString(
 				resourceBundle,
-				"this-number-refers-to-the-volume-of-people-that-find-your-" +
-					"page-through-a-search-engine")
+				"this-is-the-number-of-page-views-generated-by-people-coming-" +
+					"from-a-search-engine")
 		).put(
 			"paid",
 			ResourceBundleUtil.getString(
 				resourceBundle,
-				"this-number-refers-to-the-volume-of-people-that-find-your-" +
-					"page-through-paid-keywords")
-		).build();
-
-		Map<String, String> titleMap = HashMapBuilder.put(
-			"organic", ResourceBundleUtil.getString(resourceBundle, "organic")
+				"this-is-the-number-of-page-views-generated-by-people-that-" +
+					"find-your-page-through-google-adWords")
```

Comment:
"adWords"? 🤔

Reply:
not sure about it, I could not find an example of camel case when using plain text

Reply:
Makes sense, I will change it :)